Detectors/EMCAL/calibration/include/EMCALCalibration/EMCALChannelCalibrator.h

```diff
@@ -69,6 +70,9 @@ class EMCALChannelCalibrator : public o2::calibration::TimeSlotCalibration<o2::e
   void setIsTest(bool isTest) { mTest = isTest; }
   bool isTest() const { return mTest; }
 
+  CcdbObjectInfoVector getInfoVector() { return mInfoVector; }
+  std::vector<DataOutput> getOutputVector() { return mCalibObjectVector; }
```

Comment:
do you really want to create a copy of returned objects? These methods should be declared as `const`

Reply:
Hi @shahor02! This is now fixed.